dev-python/uvicorn/uvicorn-0.11.3.ebuild

```diff
@@ -0,0 +1,81 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+PYTHON_COMPAT=( python3_{6,7} )
+
+DISTUTILS_USE_SETUPTOOLS=rdepend
+
+inherit distutils-r1 eutils
+
+DESCRIPTION="The lightning-fast ASGI server"
+HOMEPAGE="
+	https://www.uvicorn.org
+	https://github.com/encode/uvicorn
+	https://pypi.org/project/uvicorn
+"
+SRC_URI="https://github.com/encode/${PN}/archive/${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="BSD"
+KEYWORDS="~amd64 ~x86"
+SLOT="0"
+
+IUSE="doc"
+REQUIRED_USE="x86? ( !doc )"
```

Comment:
@AndrewAmmerlaan
```[13:50] (@grknight) Baliano: REQUIRED_USE="x86? ( !doc )" is not allowed.  if an arch cannot handle something guarded by a USE, then that USE must be added to the arch's package.use.mask```
BTW why can't build doc on x86?

Reply:
I'll look into keywording it for you.

Reply:
Done, please rebase your branch against freshly synced ::gentoo repo master.

Reply:
> Done, please rebase your branch against freshly synced ::gentoo repo master.

Thank you :)

I added some the x86 keyword to some other packages in GURU that depended on mkdocs. And I got rid of the ugly workaround preventing USE="doc" on relevant packages.